src/components/wallet/PageTransactions.vue

```diff
@@ -162,6 +161,26 @@ export default {
           // loads new portion
           this.pageNumber++
           this.dataLoaded = false
+          this.$apollo.queries.transactions.fetchMore({
+            // New variables
+            variables: {
+              networkId: this.network,
+              address: this.address,
+              pageNumber: this.pageNumber
+            },
+            // Transform the previous result with new data
+            updateQuery: (previousResult, { fetchMoreResult }) => {
+              this.lastLoadedRecordsCount =
+                fetchMoreResult.transactionsV2.length
+              return {
+                __typename: previousResult.transactionsV2.__typename,
```

Comment:
this is needed?

Reply:
Which line? __typename? Yap

Reply:
why?

Reply:
I tested it and it works without it. If it is needed for some reason that I cannot see right now we can add it later.

Let's merge this! 💪

Reply:
https://vue-apollo.netlify.com/guide/apollo/pagination.html

Reply:
But it is working well... 🤷‍♀